cypress/integration/Cy_Writing_Page.tsx

```diff
@@ -0,0 +1,28 @@
+describe('Writing Page', () => {
+    beforeEach(()=> {
+        cy.visit('https://one-minute-writer.herokuapp.com/writing-page') 
+    })
+    it('show the header', () => {
+      cy.get('header').should('be.visible')
+    })
+    it('should have an inspiration section', () => {
+        cy.get('.inspirations').should('be.visible')
+        cy.get('.image-box').should('be.visible')
+            .get('img').should('be.visible')
+            .get('.new-img-btn').should('be.visible')
+        cy.get('.word-box').should('be.visible')
+            .get('.word-inspo').should('be.visible')
+            .get('.new-word-btn').should('be.visible')
+        cy.get('.audio-box').should('be.visible')
+            .get('.new-audio-btn').should('be.visible')
+    })
+    it.only('Should have a writing section', () => {
```

Comment:
Let's make sure to take out this only so that all of our tests run instead of just this one.

Reply:
Oh yes, the "it.only"